components/Events/index.js

```diff
@@ -1,55 +1,33 @@
 import styled from "styled-components";
+import { useState, useEffect } from "react";
+import useSWR from "swr";
+function formadDate(timestamp) {
+  const date = new Date(timestamp);
+  return date.toLocaleDateString();
+}
 
 export default function Events() {
+  const { data: events, error, mutate } = useSWR("/api/events");
+
+  if (!events) {
+    return "loading...";
+  }
+
   return (
     <StyledSection>
       <StyledEvents>
         <StyledEventList>
-          <li>
-            <StyledTime>
-              <h2>12</h2>
-              <p>October</p>
-            </StyledTime>
-            <StyledDetails>
-              <h3>Band: Stables</h3>
-              <p>Venue: The Fiddlers</p>
-              <p>Address: Frongasse 9, 53121 Bonn</p>
-              <p>Time: 20:00</p>
-              <a href="https://www.songkick.com/de/concerts/41025203-stables-at-fiddlers">
-                Buy tickets
-              </a>
-            </StyledDetails>
-          </li>
-          <li>
-            <StyledTime>
-              <h2>12</h2>
-              <p>September</p>
-            </StyledTime>
-            <StyledDetails>
-              <h3>Band: REMBRANDT TRIO - Netherland Jazz Classical</h3>
-              <p>Venue: Harmonie Bonn</p>
-              <p>Address: Frongasse 28-30, 53121 Bonn</p>
-              <p>Time: 19:00</p>
-              <a href="https://www.beethovenfest.de/de/programm-tickets/rembrandt-trio/163">
-                Buy tickets
-              </a>
-            </StyledDetails>
-          </li>
-          <li>
-            <StyledTime>
-              <h2>12</h2>
-              <p>August</p>
-            </StyledTime>
-            <StyledDetails>
-              <h3>Band: IAN PAICE (DEEP PURPLE) - feat. Purpendicular</h3>
-              <p>Venue: Harmonie Bonn</p>
-              <p>Address:Frongasse 28-30, 53121 Bonn</p>
-              <p>Time: 20:00</p>
-              <a href="https://shop.derticketservice.de/harmonie-bonn-tickets/details/?evid=2697725&referer_info=&tId=&forwardingTicket=">
-                Buy tickets
-              </a>
-            </StyledDetails>
-          </li>
+          {events.map((event, index) => (
+            <li key={index}>
+              <StyledTime>{formadDate(event.date)}</StyledTime>
+              <StyledDetails>
```

Comment:
I think this is currently a styled `p`. But that will cause problems because you are not allowed to nest `headings` or other `p` tags inside of it. I would consider switching it to a styled `div`.

Reply:
Done! :)